ansible/roles-infra/infra-aws-sandbox/defaults/main.yml

```diff
@@ -24,9 +24,13 @@ output_dir: '/tmp/output_dir'
 kerberos_realm: OPENTLC.COM
 ipa_domain: opentlc.com
 
-opentlc_admin_backdoor: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCvZvn+GL0wTOsAdh1ikIQoqj2Fw/RA6F14O347rgKdpkgOQpGQk1k2gM8wcla2Y1o0bPIzwlNy1oh5o9uNjZDMeDcEXWuXbu0cRBy4pVRhh8a8zAZfssnqoXHHLyPyHWpdTmgIhr0UIGYrzHrnySAnUcDp3gJuE46UEBtrlyv94cVvZf+EZUTaZ+2KjTRLoNryCn7vKoGHQBooYg1DeHLcLSRWEADUo+bP0y64+X/XTMZOAXbf8kTXocqAgfl/usbYdfLOgwU6zWuj8vxzAKuMEXS1AJSp5aeqRKlbbw40IkTmLoQIgJdb2Zt98BH/xHDe9xxhscUCfWeS37XLp75J
+opentlc_admin_backdoor: >-
+  ssh-rsa
+  AAAAB3NzaC1yc2EAAAADAQABAAABAQCvZvn+GL0wTOsAdh1ikIQoqj2Fw/RA6F14O347rgKdpkgOQpGQk1k2gM8wcla2Y1o0bPIzwlNy1oh5o9uNjZDMeDcEXWuXbu0cRBy4pVRhh8a8zAZfssnqoXHHLyPyHWpdTmgIhr0UIGYrzHrnySAnUcDp3gJuE46UEBtrlyv94cVvZf+EZUTaZ+2KjTRLoNryCn7vKoGHQBooYg1DeHLcLSRWEADUo+bP0y64+X/XTMZOAXbf8kTXocqAgfl/usbYdfLOgwU6zWuj8vxzAKuMEXS1AJSp5aeqRKlbbw40IkTmLoQIgJdb2Zt98BH/xHDe9xxhscUCfWeS37XLp75J
 
-ocpkey: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC8c3m39CoyA3bbgQUui3MGlJOryfg98NwIj8UUOv+PCuTRiyNanZeirRnPzWBtPzrVXeqtfenJo1ehDt0SRoqlUvpE7IUgnJsM7XJaUmKiVJMI4WslgMQMsZQGol7n2h0jwvtXIQp44jTtX61G7HMIUoFjJNXCO5GCrj3w2SVAKMYcznKUXpVmrkjdiHTZHmDxArHKZ7Kew2jCNUMgEq+mcaINPovUybjD17rCiKJDV4JQvLFQH89BFTVkVBsc0+QAe9LoW6l0rMDSemTvFeHy7tD0qGxYehvnHreOH6Nn9j9iOlG0f4R9+M+8KaI3uqibjOhn/kOQLK/1mphg2NnJ
+ocpkey: >-
+  ssh-rsa
+  AAAAB3NzaC1yc2EAAAADAQABAAABAQC8c3m39CoyA3bbgQUui3MGlJOryfg98NwIj8UUOv+PCuTRiyNanZeirRnPzWBtPzrVXeqtfenJo1ehDt0SRoqlUvpE7IUgnJsM7XJaUmKiVJMI4WslgMQMsZQGol7n2h0jwvtXIQp44jTtX61G7HMIUoFjJNXCO5GCrj3w2SVAKMYcznKUXpVmrkjdiHTZHmDxArHKZ7Kew2jCNUMgEq+mcaINPovUybjD17rCiKJDV4JQvLFQH89BFTVkVBsc0+QAe9LoW6l0rMDSemTvFeHy7tD0qGxYehvnHreOH6Nn9j9iOlG0f4R9+M+8KaI3uqibjOhn/kOQLK/1mphg2NnJ
```

Comment:
That looks weird

i think we can locally disable:

```
# yamllint disable rule:line-length
```
for ssh pub keys entries

Reply:
Sounds good, will do!